lib/ansible/modules/cloud/ovirt/ovirt_host_network.py

```diff
@@ -74,7 +74,7 @@
         type: bool
     save:
         description:
-            - "If I(true) network configuration will be persistent, by default they are temporarily."
+            - "If I(true) network configuration will be persistent, otherwise it is temporary. Since Ansible 2.8 I(true) is the default."
         type: bool
```

Comment:
Please add default value also into documentation

Reply:
Is this not the documentation? Would you please specify where else to make changes?

Reply:
Is the comment at the start of the file not the documentation? Would you please specify where else to make changes? thanks